### PR TITLE
Add a PR template for balance changes.

### PR DESCRIPTION
**Pull request by Quantumshark**
I think balance changes don't really fit very well into any of the three categories that we currently have - they're not new content, so Content is inappropriate, they're not really bugfixes, and they certainly aren't code features.

The structure I've provided in the template gives these sections:
1. An explanation of the change made. This should be a short summary, and may not be needed if the change is small enough that it's immediately understandable from looking under files changed.
2. An explanation for the reasons for making that change. This is the place to link relevant issues.
3. Space to attach a savefile, if appropriate. A balance change to a mission, or to a ship or outfit that only becomes available past some point in the storyline, probably should have a savefile, while a balance change which is more general and/or immediately accessible probably shouldn't.

(There's no PR template for new PR templates, either, but that's probably fine. I can make one if people want one.)

### CODE REVIEW

**Comment by lumbar527** [comment]
Looks reasonable.

> (There's no PR template for new PR templates, either, but that's probably fine. I can make one if people want one.)

(:

**Comment by quyykk** [comment]
> (There's no PR template for new PR templates, either, but that's probably fine. I can make one if people want one.)

plz no lol

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ This save file can be used to verify the bugfix. The bug will occur when using {
 {{attach a save file that can be used to verify your bugfix. It MUST have no plugin requirements}}
 
 
+-----------------------
+**Balance:**
+
+## Changes Made
```
Comment (quyykk) [inline .github/pull_request_template.md:37]:
```suggestion
## Summary
```

thoughts?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -31,6 +31,20 @@ This save file can be used to verify the bugfix. The bug will occur when using {
...
+{{add details}}
```
Comment (quyykk) [inline .github/pull_request_template.md:38]: this should be expanded upon imo

File: .github/pull_request_template.md (hunk 1 above)
Comment (lumbar527) [reply .github/pull_request_template.md:37]: Yeah, it should be `Summary`.

File: .github/pull_request_template.md (hunk 2 above)
Comment (lumbar527) [reply .github/pull_request_template.md:38]:
```suggestion
{{concisely explain what changes you made to improve balance}}
```
What about this?

Comment (Quantumshark) [reply .github/pull_request_template.md:38]: Maybe just `concisely explain the changes you made`? The part about improving balance is what the "Reasoning" section is for, that's where you explain why the changes are improving balance.

Comment (quyykk) [reply .github/pull_request_template.md:38]: `summarize all the changes you made` maybe?

Comment (lumbar527) [reply .github/pull_request_template.md:38]: I'd say @quyykk's suggestion.

**Review by quyykk — APPROVED** [review]

**Review by mOctave — APPROVED** [review]

**Pull request merged**